core/storage/email/gae_models.py

```diff
@@ -127,7 +127,7 @@ def create(
         email_model_instance.put()
 
     def put(self):
-    	"""Put the email instance into the messages by hashing it."""
+    	"""Creates the hash value of the instance of SentEmailModel."""
```

Comment:
Indentation of this line is wrong. It should be by 4 spaces from the previous line.

Also, the docstring is still incorrect. The implementation does create a hash value but that's not really the "point" of this method -- note that it calls the superclass put(). Really, it should be something like: "Saves this SentEmailModel instance to the datastore."